src/main/java/mate/controller/CreateCarController.java

```diff
@@ -0,0 +1,46 @@
+package mate.controller;
+
+import java.io.IOException;
+import java.util.NoSuchElementException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.service.CarService;
+import mate.service.ManufacturerService;
+
+@WebServlet(urlPatterns = "/cars/add")
+public class CreateCarController extends HttpServlet {
```

Comment:
```suggestion
public class AddCarController extends HttpServlet {
```
you use add verb in URL and jsp, so use it in controller's name too to keep to consistencty, the same for other Create controllers